posthog/hogql_queries/web_analytics/test/test_web_stats_table.py

```diff
@@ -1246,123 +1254,38 @@ def test_timezone_filter_general(self):
             [0, (1, None), (1, None), ""],
         ]
 
-    def test_timezone_filter_dst_change(self):
```

Comment:
dst changes aren't handled by us any more, the browser just sends the right offset